noteable/datasources.py

```diff
@@ -121,43 +121,43 @@ def bootstrap_datasource(
             metadata['allow_datasource_dialect_autoinstall'],
         )
 
+        # Prepare connection URL string.
```

Comment:
All this goes into the try / catch block so any issues will be captured into add_bootstrapping_failure().